spec/controllers/api/scores_controller_spec.rb

```diff
@@ -104,4 +104,16 @@
       expect(Score.count).to eq score_count
     end
   end
+
+  describe 'GET user scores' do
+    it 'should return all scores for a user' do
+      get api_path(@user2.id)
+
+      expect(response).to have_http_status(:ok)
+      response_hash = JSON.parse(response.body)
+      scores = response_hash['scores']
+
+      expect(scores.size).to eq 2
```

Comment:
We could also check for the scores' users to validate that they belong to `@user2`